packages/components/filters/filters-list/src/filters-list.stories.tsx

```diff
@@ -59,19 +59,31 @@ export const BasicExample: Story = () => {
     clearSecondaryColorFilter();
   };
 
+  // const appliedFilters = [
+  //   {
+  //     filterKey: 'primaryColors',
+  //     values: [{ value: 'blue', label: 'BLUE' }],
+  //   },
+  // ];
```

Comment:
if you set this value, you can see that the `primaryColors` `FilterMenu` is displayed on initial mount.